store/tikv/tikvrpc/tikvrpc.go

```diff
@@ -155,6 +159,20 @@ type Response struct {
 type CopStreamResponse struct {
 	tikvpb.Tikv_CoprocessorStreamClient
 	*coprocessor.Response // The first result of Recv()
+	lease                 TimeoutItem
+}
+
+// NewCopStreamResponse returns a CopStreamResponse.
+func NewCopStreamResponse(client tikvpb.Tikv_CoprocessorStreamClient, resp *coprocessor.Response, cancel context.CancelFunc, ch chan<- *TimeoutItem) *CopStreamResponse {
+	ret := &CopStreamResponse{
+		Tikv_CoprocessorStreamClient: client,
+		Response:                     resp,
+	}
+	ret.lease.cancel = cancel
+	// If the return value is not used within 1 minute, it will expire.
+	ret.lease.deadline = time.Now().Add(time.Minute).Unix()
```

Comment:
I think it's unnecessary. It can only handle the case that neither of `Recv()` or `Close()` is called.
If the `Recv()` is called at least once and then we forget to `Close()`, the `deadline` will also be 0 forever.

Reply:
Yes